backend/models/User.js

```diff
@@ -29,37 +29,34 @@ const UserSchema = new mongoose.Schema({
       "Administration",
       "Management",
       "Sales Rep",
-      "Sales Rep + Showroom",
-      "Sales Rep + Demoline",
-      "Sales Rep + Benefits",
       "Reception"
     ],
     required: 'User must have a role'
   },
-  permissions: {
-    type: [String],
-    enum: [
-      'Add Dealerships',
-      'View Dealerships',
-      'Edit Dealerships',
-      'Delete Dealerships',
-      'Add Staff Users',
-      'View Staff Users',
-      'Edit Staff Users',
-      'Delete Staff Users',
-      'Add Vehicles',
-      'View Vehicles',
-      'Edit Vehicles',
-      'Edit Vehicle Locations',
-      'Sell Vehicles',
-      'Delete Vehicles',
-      'Add Vehicle Property',
-      'Edit Vehicle Properties',
-      'View Vehicle Properties',
-      'Delete Vehicle Properties'
-    ],
-    required: 'User must have a list of permissions, even if that list is empty.'
-  },
+  // permissions: {
```

Comment:
I'm approving this for now but the commented code needs to be removed after testing

Reply:
Sure, that's the plan. We'd need to perform a end to end testing then we can remove this commented code.